monkey/infection_monkey/model/host.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, ip_addr, domain_name=''):
         self.services = {}
         self.monkey_exe = None
         self.default_tunnel = None
-        self.default_server = None
+        self.server_list = None
```

Comment:
We renamed this field because we are going to set it to a list of servers instead of a single server. My two questions to you (which are related) is what `VictimHost` class has to do with our change and did you also rename the usages of this property?